wordclock/displaySettings.h

```diff
@@ -15,8 +15,9 @@
 
 #define WORDCLOCK_ROWS_MAX 13 //The max values are used to allocate memory.
 #define WORDCLOCK_COLLS_MAX 13
-#define MAX_BRIGHTNESS_OFFSET  5
+#define MAX_BRIGHTNESS_OFFSET 5
 #define BRIGHTNESS_COUNT  12
+#define MAX_STRING_SIZE 32
```

Comment:
32 characters is quite small. is it a problem to make it 100? Risk for stack overflow? Flash size?

Reply:
32 paste in de huidige situatie, maar 100 prima